status-im/status-react
# Fix the channel-action when the text is too long

fixes #19438

### Summary
Fix the size of the large channel action if the text is too long

### Steps to test


check out the channel actions preview or ...

- Open Status
- open chat
- pin large message
- look at channel actions

status: ready
